src/BridgeHelper/XDBCBridgeHelper.h

```diff
@@ -63,10 +63,12 @@ class XDBCBridgeHelper : public IXDBCBridgeHelper
     XDBCBridgeHelper(
             ContextPtr context_,
             Poco::Timespan http_timeout_,
-            const std::string & connection_string_)
+            const std::string & connection_string_,
+            bool use_connection_pooling_=true)
```

Comment:
let's get rid of default value, to set it explicitly everywhere with setting.

Reply:
sure I will fix it.

Reply:
@bharatnc,  let's do? Waiting for this

Reply:
sure, working on it, I'll update the PR soon.

Reply:
fixed.